apel/parsers/sge.py

```diff
@@ -89,9 +89,9 @@ def _load_multipliers(self):
         hosts = xml_str.getElementsByTagName("host")
 
         for host in hosts:
-            host_name = dict(host.attributes.items())["name"]
+            host_name = host.getAttribute("name")
```

Comment:
What was it that you tried originally that didn't work?

Reply:
`host.attributes["name"]` didn't originally work but using the xml command getAttribute instead made the pytests succeed